modules/tinymce/src/plugins/emoticons/main/ts/api/Options.ts

```diff
@@ -40,7 +40,7 @@ const register = (editor: Editor, pluginUrl: string): void => {
 
   registerOption('emoticons_images_url', {
     processor: 'string',
-    default: 'https://twemoji.maxcdn.com/v/13.0.1/72x72/'
+    default: 'https://cdn.jsdelivr.net/gh/twitter/twemoji@14.0.2/assets/72x72/'
```

Comment:
I think I'd trust cloudflare over jsdelivr at this point.
```suggestion
    default: 'https://cdnjs.cloudflare.com/ajax/libs/twemoji/15.1.0/72x72/'
```

Cloudflare have also automatically redirected the `twemoji` link to the [new maintenance home](https://github.com/jdecked/twemoji) after twitter fired the developers working on it.